include/cmdlist_conv.hpp

```diff
@@ -285,7 +285,7 @@ class CDMPDVCmdListConvHelper : public CDMPDVCmdListKHelper {
       }
 
       if ((kcmd.z > 1) || (kcmd.run[i_run].pz > 1) ||
-          (kcmd.run[i_run].conv_dilation)) {
+          (kcmd.run[i_run].conv_dilation & 0xfefe)) {
```

Comment:
this might be incorrect,

better to modify `conv_dilation` on the command addition to be 0 in such case, like:
```
if (!(conv_dilation & 0xfefe)) {
  conv_dilation = 0;
}
```

Reply:
I reset this code.